tools/tpm2_getmanufec.c

```diff
@@ -410,15 +410,19 @@ int RetrieveEndorsementCredentials(char *b64h)
 
 int TPMinitialProvisioning(void)
 {
-    char *b64 = Base64Encode(HashEKPublicKey());
+    int rc = 1;
+    unsigned char *hash = HashEKPublicKey();
+    char *b64 = Base64Encode(hash);
     if (!b64) {
         LOG_ERR("Base64Encode returned null");
-        return 1;
+        goto out;
     }
 
     LOG_INFO("%s", b64);
 
-    int rc = RetrieveEndorsementCredentials(b64);
+    rc = RetrieveEndorsementCredentials(b64);
+out:
+    free(hash);
     free(b64);
```

Comment:
This is an unrelated change and should be in a separate patch.

Reply:
It is, it's on 0ec3a5a where as the move is on ff44794

Reply:
@williamcroberts ah, sorry. For some reasons I thought it was part of the same commit. Sorry for the noise.